autonav_ws/src/autonav_nav/src/astar.py

```diff
@@ -57,7 +87,12 @@ def onReset(self):
         best_pos = (0, 0)
         waypoints = []
 
+    def getWaypointsForDirection(self):
+        direction_index = self.config.getInt(CONFIG_WAYPOINT_DIRECTION)
+        return simulation_waypoints[direction_index] if self.getSystemState().mode == SystemMode.SIMULATION else competition_waypoints[direction_index] if self.getSystemState().mode == SystemMode.COMPETITION else practice_waypoints[direction_index]
```

Comment:
At competition, I experimented with creating several sets of waypoint arrays. Some were exactly what we were given, and some were copies that I modified to slightly adjust the waypoints. It would be nice to make that a configuration in the future to have any different competition waypoint arrays.

Reply:
Yeah that makes sense. I can definitely add that in the future